### ISSUE TITLE
Campaign sectors become available in custom games after beating

### ISSUE THREAD

**Issue by LibreBulb** [Anuken/Mindustry-Suggestions#4104]
### Describe the content or mechanics you are proposing.

When you beat a sector, it shows up in the "custom game" section. since there are a lot of sectors, they could be shown by pressing a button, as demonstrated below:
![57B38F4F-5093-4A7B-A48D-56BE29781F07](https://user-images.githubusercontent.com/106635945/201242012-7d1b0cba-60db-49b8-82c2-c006f715bcb8.png)
If there are still too many maps, they could be futher filtered into groups by planet, showing, say, only serpulo maps, and vice versa.

### Describe how you think this content will improve the game. If you're proposing new content, mention how it may add more gameplay options or how it will fill a new niche.

I think that the current stock selection of attack maps in the custom games directory is kind of bland; Maps like Glacier or Passage don't even have working waves anymore, nor are they even attack maps, as AI building was removed; they're really just PvP maps now. 

Adding campaign sectors to the custom games as you beat them is an easy way to freshen up the stock custom maps selection. 
It's also an easy way to replay all the campaign sectors repeatedly, without having to progress the campaign to get access to them (although, you will have to beat the sector at least *once*).

It can be a learning experience for new players too, as being able to replay the map to see what could have been done better, or what techniques could have been used, is a much better habit to build than mindlessly asking "HoW dO i BeAt ___" in the #mindustry channel.

**Tl;Dr:
-Good way to increase amount of stock custom maps**
*(PS fix old attack maps, now broken b/c no AI building)*

**-Easy way to replay maps w/o needing to go through campaign again**

**-Help new players learn basic critical thinking skills instead of spamming mindustry discord with   "how to beat ___ ez"**

### Before making this issue, check the boxes below to confirm that you have acknowledged them.

- [X] I have checked the [Trello](https://trello.com/b/aE2tcUwF/mindustry-trello) to make sure my suggestion isn't planned or implemented in a development version.
- [X] I am familiar with all the content already in the game or have glanced at the wiki to make sure my suggestion doesn't exist in the game yet.
- [X] I have read [README.md](https://github.com/Anuken/Mindustry-Suggestions/blob/master/README.md) to make sure my idea is not listed under the "A few things you shouldn't suggest" category.
